src/zigwin32.build.zig

```diff
@@ -1,16 +1,27 @@
 const builtin = @import("builtin");
 const std = @import("std");
 
-const is_zig_0_11 = std.mem.eql(u8, builtin.zig_version_string, "0.11.0");
+const zig_version = builtin.zig_version;
```

Comment:
If we're gonna use a switch below, we can remove this line/variable

Reply:
Sounds good!